joplin/js/CreateContentModal/ButtonBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const buttonRowClassName = (activeStep) => {
```

Comment:
maybe we could use [classnames](https://github.com/JedWatson/classnames) for this instead?

Reply:
✅